models/netsuite2/intermediate/base/int_netsuite2__accounting_periods.sql

```diff
@@ -1,27 +1,67 @@
 {{ config(enabled=var('netsuite_data_model', 'netsuite') == var('netsuite_data_model_override','netsuite2')) }}
 
 with accounting_periods as (
-
-    select *
-    from {{ var('netsuite2_accounting_periods') }}
+    select * from {{ var('netsuite2_accounting_periods') }}
```

Comment:
Unsure why the format of these select statements changed. Will update to the original format before merge.

Reply:
Corrected.